rose/server/game.py

```diff
@@ -96,9 +98,10 @@ def drive_player(self, name, info):
         self.players[name].response_time = info.get('response_time', 1.0)
 
     def print_stats(self):
-        log.info('Stats')
+        string = 'Stats' + os.linesep
         for i, p in enumerate(sorted(self.players.values())):
-            log.info('%d  %10s  row:%d  score:%d', i + 1, p.name, p.y, p.score)
+            string += '%d  %10s  row:%d  score:%d'.format(i + 1, p.name, p.y, p.score)
+        log.info(string)
```

Comment:
You should never use a format string including user input (player.name) - what if player.name includes a formatting placeholder? the logger will raise trying to format a missing argument. In the worst case, your log will be swallowed silently in runtime.